frontend/packages/ceph-storage-plugin/src/components/create-storage-system/reducer.ts

```diff
@@ -154,6 +150,8 @@ export type LocalVolumeSet = {
 export const reducer: WizardReducer = (prevState, action) => {
   const newState = _.cloneDeep(prevState);
   switch (action.type) {
+    case 'wizard/setInitialState':
```

Comment:
use constant for these

Reply:
We have types for these. These are consts only.
https://github.com/openshift/console/pull/9818#discussion_r691939868